source/_integrations/valve.mqtt.markdown

```diff
@@ -0,0 +1,372 @@
+json_attributes_topic:
+  description: The MQTT topic subscribed to receive a JSON dictionary payload and then set as sensor attributes. Usage example can be found in [MQTT sensor](/integrations/sensor.mqtt/#json-attributes-topic-configuration) documentation.
+  required: false
+  type: string
+name:
+  description: The name of the valve. Can be set to `null` if only the device name is relevant.
+  required: false
+  type: string
+  default: MQTT valve
+object_id:
+  description: Used instead of `name` for automatic generation of `entity_id`
+  required: false
+  type: string
+optimistic:
+  description: Flag that defines if switch works in optimistic mode.
+  required: false
+  type: boolean
+  default: "`false` if state or position topic defined, else `true`."
+payload_available:
+  description: The payload that represents the online state.
+  required: false
+  type: string
+  default: online
+payload_close:
+  description: The command payload that closes the valve. Is only used when `reports_position` is set to `false` (default). The `payload_close` is not allowed if `reports_position` is set to `true`. Can be set to `null` to disable the valve's close option.
+  required: false
+  type: string
+  default: CLOSE
+payload_not_available:
+  description: The payload that represents the offline state.
+  required: false
+  type: string
+  default: offline
+payload_open:
+  description: The command payload that opens the valve. Is only used when `reports_position` is set to `false` (default). The `payload_open` is not allowed if `reports_position` is set to `true`. Can be set to `null` to disable the valve's open option.
+  required: false
+  type: string
+  default: OPEN
+payload_stop:
+  description: The command payload that stops the valve. When not configured the valve will not support the `valve.stop` service.
+  required: false
+  type: string
+position_closed:
+  description: Number which represents closed position. The valves position will be scaled to (`position_closed`...`position_open`) range. When a service is called, and scaled back when a value is received.
+  required: false
+  type: integer
+  default: 0
+position_open:
+  description: Number which represents open position. The valves position will be scaled to (`position_closed`...`position_open`) range. When a service is called, and scaled back when a value is received.
+  required: false
+  type: integer
+  default: 100
+qos:
+  description: The maximum QoS level to be used when receiving and publishing messages.
+  required: false
+  type: integer
+  default: 0
+reports_position:
+  description: "Set to `true` if the value reports the position or supports setting the position. Enabling the `reports_position` option will cause the position to be published instead of a payload defined by `payload_open`, `payload_close` or `payload_stop`. When receiving messages `state_topic`, will accept numeric payloads or a state message: `open`, `opening`, `closed`, or `closing`."
```

Comment:
```suggestion
  description: "Set to `true` if the value reports the position or supports setting the position. Enabling the `reports_position` option will cause the position to be published instead of a payload defined by `payload_open`, `payload_close` or `payload_stop`. When receiving messages, `state_topic` will accept numeric payloads or one of the following state messages: `open`, `opening`, `closed`, or `closing`."
```